mcp/include/gondola_thermometry.h

```diff
@@ -0,0 +1,9 @@
+// Copyright 2023 Ian Lowe
```

Comment:
License legalese should match other files in mcp, not just defeat the static analysis tool (style checker)

Reply:
TBD